packages/core/src/App/Containers/Modals/mt5-notification/mt5-notification-description.tsx

```diff
@@ -5,56 +5,39 @@ import { Localize } from '@deriv/translations';
 const MT5NotificationDescription = () => {
     return (
         <div className={'mt5-notification-modal-description'}>
-            <Text as='p' size='xs'>
-                <Localize i18n_default_text='Follow these simple steps to fix it:' />
-            </Text>
             <ol className='mt5-notification-list-container'>
-                <li className='mt5-notification-list-container-item'>
-                    <Text as='p' size='xs'>
-                        <Localize i18n_default_text='On your MT5 mobile app, delete your existing Deriv account:' />
-                    </Text>
-                    <ul>
-                        <li className='mt5-notification-list'>
-                            <Text as='p' size='xs'>
-                                <Localize
-                                    i18n_default_text='- iOS: Swipe left on the account and tap <0>Delete</0>.'
-                                    components={[<strong key={0} />]}
-                                />
-                            </Text>
-                        </li>
-                        <li className='mt5-notification-list'>
-                            <Text as='p' size='xs'>
-                                <Localize
-                                    i18n_default_text='- Android: Tap the account, open <0>Options</0>, and tap <0>Delete</0>.'
-                                    components={[<strong key={0} />]}
-                                />
-                            </Text>
-                        </li>
-                    </ul>
-                </li>
-                <li className='mt5-notification-list-container-item'>
+                <li className='mt5-notification-list'>
                     <Text as='p' size='xs'>
                         <Localize
-                            i18n_default_text='Search for the broker <0>Deriv Holdings (Guernsey) Limited</0> and select it.'
-                            components={[<strong key={0} />]}
+                            i18n_default_text='Log back in to MT5 after 7:30 GMT on 20 Oct 2023 if you’re having difficulty logging in to MT5 as we’re making some updates to our MT5 platform. <0>Follow these steps</0> to log back in to MT5.'
+                            components={[
```

Comment:
This needs to be a push notification from the BE side! why are we hard coding these details in the FE code?

Reply:
This needs to be from BE but unfortunately it is high priority so we have to move forward this time as an exception going forward we are not going to pick these types of cards.
cc: @hirad-deriv @mahdiyeh-deriv

Reply:
As Hamza mentioned, the process of having our requirements released takes a long time and since this is high priority we cannot wait for those changes